app/helpers/reactjs_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ReactjsHelper
-  def mount_react_component(name, selector, data)
+  def mount_react_component(name, selector, data = [])
```

Comment:
I'm curious, is there any reason not to pass the data directly when mounting the component instead of having to read it from the `data-flash` attribute afterwards?